index.js

```diff
@@ -1026,26 +1026,32 @@ class protractorImageComparison {
      * @param {object} options (non-default) options
      * @param {int} options.resizeDimensions the value to increase the size of the element that needs to be saved
      * @param {boolean} options.disableCSSAnimation enable or disable CSS animation
+     * @param {boolean} options.canvasScreenshot enable or disable taking screenshot directly from canvas (via dataUrl instead of browser.takeScreenshot())
      * @returns {Promise} The images has been saved when the promise is resolved
      * @public
      */
-    saveElement(element, tag, options) {
+    async saveElement(element, tag, options) {
```

Comment:
Currently I don't want to use `async / await` due to the support of nodejs for this module. This would be breaking for a lot of people. Secondly if we are going to use it we need to implement it in the complete library